src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx

```diff
@@ -40,7 +40,12 @@ export class Carousel extends React.Component<Props> {
     const { sources } = this.props
 
     return sources.map((source, i) => (
-      <Flex key={this.keyForSource(source)} mr={1} alignItems="flex-start">
+      <Flex
+        key={this.keyForSource(source)}
+        mr={i === sources.length - 1 ? 2 : 1}
+        pr={i === sources.length - 1 ? 2 : 0}
```

Comment:
We repeat this logic twice: `i === sources.length - 1`. Maybe we could put it in a local var? It'd make it easier to read.

Reply:
Makes sense, just updated